### PR TITLE
Dev: doc: Mention /etc/crm/profiles.yml in man crm

### PR DESCRIPTION
**Pull request by liangxin1300**
see #901, mention /etc/crm/profiles.yml(see #842) in man crm

### CODE REVIEW

**Comment by liangxin1300** [comment]
@tomschr @zzhou1 Please review this when you have time
Thanks!

File: doc/profiles.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+==== How the content of the file is interpreted
+
+Profiles are only loaded on bootstrap init node.
+
+"default" profile is loaded in the beginning.
+
+Those specific profile will override the corresponding values in "default" profile if the specific environment is detected.
+
+Users could customize the "default" profile for their needs, for example, those on-premise environments which is not defined yet.
```
Comment (tomschr) [inline doc/profiles.adoc:48]: I would suggest to list these "properties" in a list:

```suggestion
The profiles has the following properties:

* Profiles are only loaded on bootstrap init node.
* The "default" profile is loaded in the beginning.
* Specific profiles will override the corresponding values in the "default" profile (if the specific environment is detected).
* Users could customize the "default" profile for their needs. For example, those on-premise environments which is not defined yet.
```

**Review by tomschr — CHANGES_REQUESTED** [review]
Thanks @liangxin1300 for your effort. Much appreciated!

I have some small ideas and suggestion. Maybe you find them useful.

File: doc/profiles.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
+=== /etc/crm/profiles.yml
+
+==== Purpose
+
+YAML file /etc/crm/profiles.yml contains corosync/sbd/pacemaker parameters for different platforms,
```
[the remark this answers is not in the document]
Comment (liangxin1300) [reply doc/profiles.adoc]: Done

File: doc/profiles.adoc (hunk 1 above)
Comment (liangxin1300) [reply doc/profiles.adoc]: Done

File: doc/profiles.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  key_name: value
+............
+
+The valid profile names are:
+"microsoft-azure", "google-cloud-platform", "amazon-web-services", "s390", "default"
+
+key_name should be known corosync/sbd/pacemaker parameters, like:
+"corosync.totem.token", "sbd.watchdog_timeout"
+
+More detects please reference corosync.conf(5) and sbd(8)
```
[the remark this answers is not in the document]
Comment (liangxin1300) [reply doc/profiles.adoc]: Done

[the remark this answers is not in the document]
Comment (liangxin1300) [reply doc/profiles.adoc]: Done

File: doc/profiles.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
+=== /etc/crm/profiles.yml
+
+==== Purpose
+
+YAML file /etc/crm/profiles.yml contains corosync/sbd/pacemaker parameters for different platforms,
+
+crmsh bootstrap detects environment and load the corresponding parameters predefined in this file.
+
+==== Synatx
```
[the remark this answers is not in the document]
Comment (liangxin1300) [reply doc/profiles.adoc]: Done

File: doc/profiles.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
+=== /etc/crm/profiles.yml
+
+==== Purpose
+
+YAML file /etc/crm/profiles.yml contains corosync/sbd/pacemaker parameters for different platforms,
+
+crmsh bootstrap detects environment and load the corresponding parameters predefined in this file.
```
[the remark this answers is not in the document]
Comment (liangxin1300) [reply doc/profiles.adoc:7]: system's environment, or, system's type

File: doc/profiles.adoc [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+............
+profile_name:
+  key_name: value
+............
+
+The valid profile names are:
+"microsoft-azure", "google-cloud-platform", "amazon-web-services", "s390", "default"
+
+key_name should be known corosync/sbd/pacemaker parameters, like:
+"corosync.totem.token", "sbd.watchdog_timeout"
```
[the remark this answers is not in the document]
Comment (liangxin1300) [reply doc/profiles.adoc]: Done

**Comment by liangxin1300** [comment]
> Thanks @liangxin1300 for your effort. Much appreciated!
> 
> I have some small ideas and suggestion. Maybe you find them useful.

Thanks for these nice suggestions!

File: doc/profiles.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply doc/profiles.adoc:7]: Maybe use "system environment" in that case to make it clear.

**Review by zzhou1 — APPROVED** [review]

**Pull request merged**